generator/profile.go

```diff
@@ -0,0 +1,34 @@
+package generator
+
+import (
+	"github.com/docker/oscalkit/types/oscal/profile"
+)
+
+//AppendAlterations appends alter attributes from import chain
```

Comment:
```suggestion
// AppendAlterations appends alter attributes from import chain
```

Reply:
Ah, missed this file. Apologies. Just pushed